modules/react-native-status/nodejs/status.cpp

```diff
@@ -354,38 +354,6 @@ void _StopCPUProfiling(const FunctionCallbackInfo<Value>& args) {
 
 }
 
-void _Identicon(const FunctionCallbackInfo<Value>& args) {
```

Comment:
Curious to know @alwx did you nuke this manually ? Asking because this file is autogenerated :)